configs/temporal.json

```
@@ -7,34 +7,114 @@
-      "page_rank": 998
+      "page_rank": 999
     },
     {
       "url": "https://docs.temporal.io/docs/concepts/activities",
-      "page_rank": 997
+      "page_rank": 999
     },
     {
       "url": "https://docs.temporal.io/docs/concepts/events",
-      "page_rank": 996
+      "page_rank": 999
     },
     {
       "url": "https://docs.temporal.io/docs/concepts/queries",
-      "page_rank": 995
+      "page_rank": 999
     },
     {
       "url": "https://docs.temporal.io/docs/concepts/task-queues",
+      "page_rank": 999
+    },
+    {
+      "url": "https://docs.temporal.io/docs/glossary",
+      "page_rank": 999
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/introduction",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/quick-install",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/production-deployment",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/server-architecture",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/versions-and-dependencies",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/namespaces",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/workflow-search",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/elasticsearch-setup",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/options",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/security",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/configuration",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/server/archive-data",
+      "page_rank": 998
+    },
+    {
+      "url": "https://docs.temporal.io/docs/go/introduction",
+      "page_rank": 997
+    },
+    {
+      "url": "https://docs.temporal.io/docs/java/introduction",
+      "page_rank": 996
+    },
+    {
+      "url": "https://docs.temporal.io/docs/php/introduction",
+      "page_rank": 995
+    },
+    {
+      "url": "https://docs.temporal.io/docs/node/introduction",
+      "page_rank": 994
+    },
+    {
+      "url": "https://docs.temporal.io/docs/system-tools/tctl",
       "page_rank": 994
     },
     {
       "url": "https://docs.temporal.io/",
+      "page_rank": 2
+    }
+    {
+      "url": "https://docs.temporal.io/blog",
       "page_rank": 1
     }
   ],
   "sitemap_urls": [
     "https://docs.temporal.io/sitemap.xml"
   ],
   "sitemap_alternate_links": true,
-  "stop_urls": [],
+  "stop_urls": [
+    "https://docs.temporal.io/blog/tags/announcement/",
+    "https://docs.temporal.io/blog/tags/case-study/",
+    "https://docs.temporal.io/blog/page/2/",
+  ],
```

```suggestion
  "stop_urls": [
    "https://docs.temporal.io/blog/tags/announcement/",
    "https://docs.temporal.io/blog/tags/case-study/",
    "https://docs.temporal.io/blog/tags/"
  ],
```

spurious tags we dont really need

ideally i'd like Algolia to just not index ANY tags.